data/core/editor/items.cfg

```diff
@@ -174,6 +241,18 @@
         name= _ "Cage"
     [/item]
 
+    [item]
+        id=chest
+        image=items/chest.png
+        name= _ "Closed Chest"
+    [/item]
+
+    [item]
+        id=chest-open
+        image=items/chest-open.png
+        name= _ "Open Chest"
```

Comment:
Could use a po hint that "Open" is an adjective, this isn't a menu option to open a chest.